README.md

```diff
@@ -65,7 +65,19 @@ for `tslint@^5.0.0`
 }
 ```
 
-**NOTE**: To use this plugin, it'd better to also use [tslint-config-prettier](https://github.com/prettier/tslint-config-prettier) to disable all prettier-related rules, so as to avoid conflicts between existed rules.
+## When to use `tslint-plugin-prettier`
```

Comment:
I think it'd be better to put them in a FAQ section, and focus on the difference between `tslint-plugin-prettier` and `tslint-config-prettier` with these scenarios provided. What do you think?